en/08_Changelogs/6.0.0.md

```diff
@@ -29,6 +30,242 @@ title: 6.0.0 (unreleased)
+-use SilverStripe\Dev\DevelopmentAdmin;
++use SilverStripe\PolyExecution\PolyOutput;
+-use SilverStripe\Security\Permission;
+ use SilverStripe\Security\PermissionProvider;
+-use SilverStripe\Security\Security;
++use Symfony\Component\Console\Command\Command;
++use Symfony\Component\Console\Input\InputInterface;
++use Symfony\Component\Console\Input\InputOption;
+
+-class MyOtherActionController extends Controller implements PermissionProvider
++class MyOtherActionController extends DevCommand implements PermissionProvider
+ {
++    protected static string $commandName = 'app:my-other-action';
++
++    protected static string $description = 'Perform my custom action in dev/my-other-action or via sake app:my-other-action';
++
++    private static array $permissions_for_browser_execution = [
++        'MY_CUSTOM_PERMISSION',
++    ];
++
++    public function getTitle(): string
++    {
++        return 'My other action';
++    }
++
+-    protected function init(): void
+-    {
+-        parent::init();
+-
+-        if (!$this->canInit()) {
+-            Security::permissionFailure($this);
+-        }
+-    }
+-
+-    public function index(HTTPRequest $request)
++    protected function execute(InputInterface $input, PolyOutput $output): int
+     {
+-        $someVar = $request->getVar('some-var');
++        $input->getOption('some-var');
+-        if (Director::is_cli()) {
+-            $body = "some output\n";
+-        } else {
+-            $body = "some output<br>\n";
+-        }
++        $output->writeln('some output');
+-
+-        return $this->getResponse()->setBody($body);
++        return Command::SUCCESS;
+     }
+
++    public function getOptions(): array
++    {
++        return [
++            new InputOption('some-var', null, InputOption::VALUE_NONE, 'some get variable'),
++        ];
++    }
+-    public function canInit(): bool
+-    {
+-        return (
+-            Director::isDev()
+-            || (Director::is_cli() && DevelopmentAdmin::config()->get('allow_all_cli'))
+-            || Permission::check('MY_CUSTOM_PERMISSION')
+-        );
+-    }
+
+     // ...
+ }
+```
+
+You would now access the `/dev/my-action` action via an HTTP request only. The `/dev/my-other-action` action can be access via an HTTP request, or by using `sake app:my-other-action` on the command line.
+
+The `some-var` option can be used in a query string when running the action via an HTTP request, or as a flag (e.g. `sake app:my-other-action --some-var`) in CLI.
+
+See [`PolyCommand`](/developer_guides/cli/polycommand#dev-commands) for more details about the `DevCommand` API.
+
+#### `sake -start` and `sake -stop` have been removed {#cli-daemon}
+
+Sake used to have functionality to make daemon processes for your application. This functionality was managed with `sake -start my-process` and `sake -stop my-process`.
+
+We've removed this functionality. Please use an appropriate daemon tool such as `systemctl` to manage these instead.
+
```

Comment:
Need to mention `dont_populate` being renamed to `no-populate`

Reply:
That's not related to `sake -start` and `sake -stop`, but I've added a note about it to the relevant place.